packages/components/MenuButton/macos/MenuButton.swift

```diff
@@ -64,16 +64,17 @@ class MenuButton: NSPopUpButton {
     }
 
     for (index, menuItem) in menu.items.enumerated() {
-      menuItem.target = self
-      menuItem.action = #selector(sendOnItemClickEvent)
       if let submenu = menuItem.submenu {
         //Add actions to one level of submenu items to support the `onSubmenuItemClick` callback
         for subMenuItem in submenu.items {
           subMenuItem.tag = index //store the index of the "super" menuItem for lookup in the action
           subMenuItem.target = self
           subMenuItem.action = #selector(sendOnSubItemClickEvent)
         }
-      }
+			} else {
```

Comment:
what is going on with your tabs?

Reply:
Damn I merged this too quickly. My xcode defaults to 4 spaces for tabs and github defaults to 8 I think..

Reply:
How hard would it be to just copy FUA's SwiftLint file and add a check to the Apple PR?